spell/Sean vas Terra; Elementalist's Pocketbook.json

```diff
@@ -880,8 +880,8 @@
 				"v": true,
 				"s": true,
 				"m": {
-					"text": "a 5-foot copper rod worth at least",
-					"cost": 1000
+					"text": "a 5-foot copper rod worth at least 10gp",
+					"cost": 10
```

Comment:
1000 is correct; it's written in copper pieces

Reply:
Completely forgot that. Fixed it.